pipeline/transformer/transformer_calm/src/main/scala/weco/pipeline/transformer/calm/transformers/CalmMergeCandidates.scala

```diff
@@ -0,0 +1,42 @@
+package weco.pipeline.transformer.calm.transformers
+
+import weco.catalogue.internal_model.identifiers.{
+  IdState,
+  IdentifierType,
+  SourceIdentifier
+}
+import weco.catalogue.internal_model.work.MergeCandidate
+import weco.catalogue.source_model.calm.CalmRecord
+import weco.pipeline.transformer.calm.models.CalmRecordOps
+
+object CalmMergeCandidates extends CalmRecordOps {
+  def apply(record: CalmRecord): List[MergeCandidate[IdState.Identifiable]] =
+    sierraMergeCandidate(record).toList ++ miroMergeCandidates(record)
+
+  private def miroMergeCandidates(record: CalmRecord) =
+    // The internal field "Wheels" is mapped to "MiroID"
+    record.getList("Wheels").map { miroIdentifier =>
+      MergeCandidate(
+        IdState.Identifiable(
+          SourceIdentifier(
+            identifierType = IdentifierType.MiroImageNumber,
+            ontologyType = "Work",
+            value = miroIdentifier
+          )
+        )
+      )
```

Comment:
Can we put a reason on these MergeCandidates?

(Currently the "reason" is an `Option[String]`, but this is currently the only transformer where we don't set one. It feels like the reason is more and more useful as merging gets more complicated, and requiring it in the type system would be a way to keep us honest.

Changing the MergeCandidate model would be a breaking change to internal model – anything that came through the old Calm transformer would have a `null` – but it opens the door to it in a future update.)

Reply:
Forgot that there was a `reason` field, will add it